persist/src/main/resources/db/seed/reservations.json

```diff
@@ -0,0 +1,5653 @@
+{
+  "reservations": [
+    {
+      "blockId": 439,
+      "userId": 47.0,
```

Comment:
Weird that these ids have `.0` but as long as you tested that this doesn't mess anything up it's fine

Reply:
Yeah idk why either but it doesn't break anything